README.md

```diff
@@ -1,118 +1,69 @@
-- [Custom Scalar Types](#custom-scalar-types)
-- [Support For Cached Responses](#support-for-cached-responses)
-- [RxJava2 Support](#rxjava2-support)
-- [Coroutines Support](#coroutines-support)
-- [Android Espresso Support](#android-espresso-support)
-- [Use generated models only](#use-generated-models-only)
-- [Gradle Configuration of Apollo Android](#gradle-configuration-of-apollo-android)
-  - [Optional Support](#optional-support)
-  - [Semantic Naming](#semantic-naming)
-  - [Java Beans Semantic Naming for Accessors](#java-beans-semantic-naming-for-accessors)
-  - [Explicit Schema location](#explicit-schema-location)
-  - [Kotlin model generation](#kotlin-model-generation)
-  - [Transformed queries](#transformed-queries)
-  - [Incubating plugin](#incubating-plugin)
-- [License](#license)
-
-<!-- END doctoc generated TOC please keep comment here to allow auto update --> 
-
-## Adding Apollo to your Project
+Apollo-android features:
+
+* Automatic generation of typesafe models.
+* Support for Java and Kotlin code generation.
+* Queries, Mutations and Subscriptions.
+* Reflection-free parsing of responses.
+* HTTP cache.
+* Normalized cache.
+* File Upload.
+* Custom scalar types.
+* Support for RxJava2 and Coroutines. 
+
+## Adding Apollo-Android to your Project
 
 The latest Gradle plugin version is [ ![Download](https://api.bintray.com/packages/apollographql/android/apollo-gradle-plugin/images/download.svg) ](https://bintray.com/apollographql/android/apollo-gradle-plugin/_latestVersion)
 
-To use this plugin, add the dependency to your project's build.gradle file:
+To use this plugin, add the dependency to your project's root build.gradle file:
 
 ```groovy
 buildscript {
   repositories {
     jcenter()
   }
   dependencies {
-    classpath 'com.apollographql.apollo:apollo-gradle-plugin:x.y.z'
+    classpath("com.apollographql.apollo:apollo-gradle-plugin:x.y.z")
   }
 }
 ```
 
-Then add the following to your app's build.gradle dependencies:
-
-```groovy
-repositories {
-    jcenter()
-}
-
-dependencies {
-  implementation 'com.apollographql.apollo:apollo-runtime:x.y.z'
-}
-```
-
-Latest development changes are available in Sonatype's snapshots repository:
+Then add the dependencies to your app's build.gradle and apply file and apply the `com.apollographql.android` plugin:
 
 ```groovy
-buildscript {
-  repositories {
-    jcenter()
-    maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
-  }
-  dependencies {
-    classpath 'com.apollographql.apollo:apollo-gradle-plugin:1.2.2-SNAPSHOT'
-  }
-}
-```
+apply plugin: 'com.apollographql.android'
 
-```groovy
 repositories {
     jcenter()
-    maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
 }
 
 dependencies {
-  implementation 'com.apollographql.apollo:apollo-runtime:1.2.2-SNAPSHOT'
+  implementation("com.apollographql.apollo:apollo-runtime:x.y.z")
+  
+  // The code generated by the plugin uses jetbrains annotations, so you will need 
```

Comment:
I don't think this is a hard requirement. What about something like `you may need to include this as compile time dependency if it has not been already available (i.e Kotlin projects already have it)`?

Reply:
Wansn't sure about that one, I changed the wording a bit.